client/src/components/App.js

```diff
@@ -14,20 +15,29 @@ const App = () => {
   return (
     <div>
       <Router history={history}>
-        <Header/>
-        <div style={{display: "flex"}}>
-          <SideNav/>
+        <Header />
+        <div style={{ display: "flex" }}>
+          <SideNav />
           <div className="ui container">
-              <div>
-                <Switch>
-                  <Route path="/" exact component={StreamList} />
-                  <Route path="/streams/new" exact component={StreamCreate} />
-                   <Route path="/streams/edit/:id" exact component={StreamEdit} />
-                  <Route path="/streams/delete/:id" exact component={StreamDelete} />
-                  <Route path="/streams/:id" exact component={StreamShow} />
-                  <Route path="/profile/:id" exact component={ProfileModal} />
-                </Switch>
-              </div>
+            <div>
+              <Switch>
+                <Route path="/" exact component={StreamList} />
+                <Route path="/streams/new" exact component={StreamCreate} />
+                <Route path="/streams/edit/:id" exact component={StreamEdit} />
+                <Route
```

Comment:
Is there a reason these Routes are formatted differently than the other ones? Were they just too long?

Reply:
It was my auto formatter that formatted it. I don't think they were formatted properly before